packages/svelte/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@serveside/ssr-aas-svelte",
+  "version": "0.0.1-alpha.4",
+  "description": "> TODO: description",
+  "author": "Moshe Uminer <47164590+mosheduminer@users.noreply.github.com>",
+  "homepage": "https://github.com/serveside/ssr-aas#readme",
+  "license": "MIT",
+  "main": "dist/core.js",
```

I just fixed this in main for the other packages, make sure to update

```suggestion
  "main": "dist/index.js",
  "publishConfig": {
    "access": "public"
  },
```